engine/orchestration/src/main/java/com/cloud/agent/manager/ClusteredAgentManagerImpl.java

```diff
@@ -368,9 +368,10 @@ public void reconnect(final long hostId) throws CloudRuntimeException, AgentUnav
         if (result == null) {
             super.reconnect(hostId);
 
```

Comment:
Instead of adding an `else` block and then an `if` nested, what about simply using a `return;` statement here?